src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -41,6 +42,27 @@ import scala.util.Try
  *    is backend-specific
  */
 
+/**
+ * Trait to encapsulate whether an execution is complete and if so provide a result.  Useful in conjunction
+ * with the `poll` API to feed results of previous job status queries forward.
+ */
+trait ExecutionHandle {
+  def isDone: Boolean
+  def result: ExecutionResult
+}
+
+/** A handle that represents a completed execution. */
```

Comment:
This comment seems not worth the effort considering it's the name of the thing for the most part :)